arbor/fvm_layout.cpp

```diff
@@ -1111,12 +1113,20 @@ fvm_mechanism_data fvm_build_mechanism_data(const cable_cell_global_properties&
         config.reset_econc.resize(n_cv);
         config.init_revpot.resize(n_cv);
 
-        cable_cell_ion_data ion_data = *(value_by_key(dflt.ion_data, ion) | value_by_key(global_dflt.ion_data, ion));
-        double dflt_iconc = ion_data.init_int_concentration;
-        double dflt_econc = ion_data.init_ext_concentration;
-        double dflt_rvpot = ion_data.init_reversal_potential;
+        auto ion_data = value_by_key(global_dflt.ion_data, ion).value();
+        auto dflt_iconc = ion_data.init_int_concentration.value();
+        auto dflt_econc = ion_data.init_ext_concentration.value();
+        auto dflt_rvpot = ion_data.init_reversal_potential.value();
 
-        const mcable_map<initial_ion_data>& ion_on_cable = initial_ion_data_map[ion];
+        if (auto ion_data = value_by_key(dflt.ion_data, ion)) {
```

Comment:
Because there is already something called `ion_data`, this is a bit confusing. Can the other `ion_data` be called `global_ion_data` instead?

Reply:
I missed that. I'll rename the previous one, thanks!